llmfoundry/models/layers/blocks.py

```diff
@@ -26,6 +26,7 @@ def __init__(
         norm_type: str = 'low_precision_layernorm',
         fc_type: str = 'torch',
         device: Optional[str] = None,
+        no_bias: bool = False,
```

Comment:
why do we use `no_bias` here but `bias` elsewhere?

Reply:
Our original config uses `no_bias`. I want to switch to `bias` everywhere since that is what everyone else does. But changing in cfg (which is passed directly into block) is a breaking change. Imo we should just do breaking change since we're on an early version of foundry...

Reply:
We can't break API bc it breaks ckpts. :(